app/models/concerns/sufia/ability.rb

```diff
@@ -52,6 +52,7 @@ def editor_abilities
 
       can :create, TinymceAsset
       can [:create, :update], ContentBlock
+      can :edit, ::SolrDocument
```

Comment:
Personally, I'd say remove lines 53-55 and just have `can :manage, :all` and be done with it. The implementor can override the method in their own `ability.rb`.